vaadin-testbench-core/src/main/java/com/vaadin/karibu/BaseKaribuTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (C) 2020 Vaadin Ltd
```

Comment:
```suggestion
 * Copyright (C) 2022 Vaadin Ltd
```
Should the year be updated to current year at least for the new headers?